Sources/SwiftSyntax/AbsolutePosition.swift

```diff
@@ -0,0 +1,59 @@
+//===--------------- AbsolutePosition.swift - Source Positions ------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// An absolute position in a source file as text - the absolute utf8Offset from
+/// the start, line, and column.
+public final class AbsolutePosition {
```

Comment:
I think it would make sense to also migrate `AbsolutePosition` to the state on `master` that is a `struct` and interacts with `SourceLength`, i.e. cherry-pick https://github.com/apple/swift-syntax/commit/fc313fc29c33baf1d56fc0c965964c2e7c517e50 and maybe https://github.com/apple/swift-syntax/commit/86be68fec5332dbde10b720bce196ed933b515a7

Reply:
Done.